easybuild/easyconfigs/e/elbencho/elbencho-2.2-5-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,39 @@
+# #
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Copyright::   HPCNow!
+# Authors::     Maicon Faria <maicon.faria@hpcnow.com>
+# License::     GPL-v3.0
+# #
+
+easyblock = 'MakeCp'
+
+name = 'elbencho'
+version = '2.2-5'
+
+homepage = 'https://github.com/breuner/elbencho/r'
+description = "elbencho was inspired by traditional storage benchmark tools like fio, mdtest and ior, but was written"\
+              "from scratch to replace them with a modern and easy to use unified tool for file systems, object stores"\
+              "block devices."
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+source_urls = ['https://github.com/breuner/elbencho/archive/refs/tags/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['4b598639452665a8b79c4c9d8a22ae63fb9b04057635a45e686aa3939ee255b4']
+
+dependencies = [
```

Comment:
The other complaint from the unit tests is about missing `binutils` as a build dependency.
That's required if you're at the `GCCcore` level.
If you do change to `GCC`, then that error should be fixed.